CorsixTH/Luatest/spec/system/path_spec.lua

```diff
@@ -20,13 +20,11 @@ SOFTWARE. --]]
 
 require("class_test_base")
 
-require("system.path_manager")
+require("system/path")
 
 describe("Path Manager -", function()
-    it("should return right separator", function()
-        local path_manager = PathManager()
-        local expected_separator = package.config:sub(1, 1)
-        assert.equal(expected_separator, path_manager:getSeparator())
-    end)
-end)
-  
\ No newline at end of file
+  it("should return right separator", function()
+    local expected_separator = package.config:sub(1, 1)
+    assert.equal(expected_separator, Path.getSeparator())
+  end)
+end)
```

Comment:
Still a problem with your file endings on all files - strange editor.
There should be a single newline at the end of the file.

Reply:
I guess it should be fixed now?